tsconfig.json

```diff
@@ -2,9 +2,9 @@
   "compilerOptions": {
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
+    "target": "es2017" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */,
     "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    // "lib": [],                             /* Specify library files to be included in the compilation. */
+    "lib": ["dom", "es2017", "esnext.asynciterable"],                             /* Specify library files to be included in the compilation. */
```

Comment:
Since you've added `esnext.asynciterable` here, you don't need it in `functions/tsconfig.json`, since that extends this file.

Reply:
Same with the `dom` inclusion. It works during first build. But when running `tsc --watch` the build will fail due to what I assume to be an issue with incldues and `tsc --watch`